src/runtime/components/ScriptGoogleAdsense.vue

```diff
@@ -14,7 +14,6 @@ const props = withDefaults(defineProps<{
    */
   trigger?: ElementScriptTrigger
 }>(), {
-  dataAdFormat: 'auto',
```

Comment:
I'm okay with this change but as it's a breaking change it would need to wait until 0.9.0 so maybe better to have it in a separate PR.

Reply:
Alright let me revise the PR